src/Pos/Explorer/Txp/Toil/Logic.hs

```diff
@@ -202,6 +202,9 @@ updateAddrBalances (combineBalanceUpdates -> updates) = mapM_ updater updates
     updater :: (MonadTxExtra m, WithLogger m) => (Address, (Sign, Coin)) -> m ()
     updater (addr, (Plus, coin)) = do
         currentBalance <- fromMaybe (mkCoin 0) <$> getAddrBalance addr
+        when (coinToInteger currentBalance + coinToInteger coin > coinToInteger maxBound) $ do
+            !() <- traceM $ sformat ("Overflow will happen soon, new balance for "%build%" is the sum of "%coinF%" and "%coinF) addr currentBalance coin
```

Comment:
This `traceM` is redundant in presence of `logError` (and, if I'm not mistaken, it triggers a warning which prevents CI to pass)